src/Microsoft.Azure.SignalR.Common/Utilities/RestClient.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.Azure.SignalR.Common;
 using Microsoft.Extensions.Primitives;
 using Newtonsoft.Json;
+using Microsoft.Azure.SignalR.Models;
```

Comment:
Is it used?

Reply:
Yes, it's used by [Payload class](https://github.com/Azure/azure-signalr/blob/00b898138dd6e0047ba174c4bf034e7c94201b30/src/Microsoft.Azure.SignalR.Common/Utilities/RestClient.cs#L128). In the past, Payload is under Microsoft.Azure.SignalR.Models. Now Payload class is generated, so the namespace is changed.